src/api/uploads/BaseUpload.js

```diff
@@ -49,6 +52,12 @@ class BaseUpload extends Base {
             url = url.replace('content', `${this.fileId}/content`);
         }
 
+        // Add unreachable hosts to url
+        const unreachableHostUrls = this.uploadsReachability.getUnreachableHostsUrls();
+        if (unreachableHostUrls.length !== 0) {
+            url += `?unreachable_hosts=${unreachableHostUrls.join(',')}`;
```

Comment:
Yes, this param is not used anywhere else. Given that I don't know how much value we would get from replacing any pre-existing params. 

The preflight call already knows how to handle this query parameter and it would only work for customers who have the Box Accelerator feature enabled.

Reply:
for persisting query params, are there other query params that could come through other than unreachable hosts? because if so, we want to persist them as well as add the new unreachable_hosts.

we are always running this reachability test. is there some way for us to identify that this is a customer who runs Box Accelerator in order to determine if we need to do reachability testing? or should we do reachability testing for all uploads?

Reply:
Don't quite understand your question about the query params. The base `url` to make the preflight request is created  just from the line above and it doesn't have any query params. As far as I'm aware there are no other query params.

There is no way for us to identify if a customer has the accelerated uploads option enabled. We will always try to do a reachability test, but for the majority of the customers we will never end up actually performing the test since the preflight request for customers who don't have the accelerator option enabled will just return the default upload host. We don't do a reachability test if the `upload_url` from the preflight request is the default upload host.

Reply:
ohh gotcha, didn't see the url line above. that makes sense.